src/base/TransactionQueue.ts

```diff
@@ -423,9 +453,23 @@ export class TransactionQueue<
    *   are sufficient to cover this queue's fees
    */
   private async assertFeesCovered(): Promise<void> {
-    const breakdown = await this.getMinFees();
+    const breakdown = await this._getMinFees();
+
+    const { accountBalance, accountFees, thirdPartyFees, subsidizedTransactions } = breakdown;
 
-    const { accountBalance, accountFees, thirdPartyFees } = breakdown;
+    const unsupportedTransactions = subsidizedTransactions.filter(
```

Comment:
Since there is no further need of unsupportedTransactions, IMO we should use `.some` instead of `.filter`. In that case we can directly check the boolean value instead of checking the length

Reply:
It's being used as part of the error data